File: cpp/bc2551.cpp
#include <iostream>

using namespace std;

int main()
{
	int cTe, dis, tMi, vel;
	double vMe;
	
	while(cin >> cTe)
	{
		vMe = 0.0;
		
		for(int i = 0; i < cTe; i++)
		{
			cin >> tMi >> dis;
			
			vel = (double) (dis*1000) / (tMi*60);
			
			if(vel > vMe)
			{
				cout << i+1 << endl;
				vMe = vel;
			}
		}
	}
}

File: python/bc2551.py
while True:
    try:

        nCa = int(input())
        vMe = 0.0

        for c in range(0, nCa):
            cVal = input().split(" ")

            tMi = int(cVal[0])
            dis = int(cVal[1])

            vel = float((dis*1000) / (tMi*60))

            if vel > vMe:
                print(c+1)
                vMe = vel
    except EOFError:
        break